assets/scripts/utils/array.js

```diff
@@ -1,101 +1,124 @@
-import { isArray } from './is';
+import { isArray } from './is'
 
-export function addToArray ( array, value ) {
-    const index = array.indexOf( value );
 
-    if ( index === -1 ) {
-        array.push( value );
+/**
+ * Add item of array if doesn't exist
+ * @param {array}       array   - array of items
+ * @param {?string}     value   - value to add to the array
+ * @return {void}
+ */
+
+const addToArray = (array, value) => {
+    if (array.indexOf(value) === -1) {
+        array.push(value)
     }
 }
```

Comment:
Should this be replaced with `Set`?

```js
const set = new Set();

set.add(1) // Set [ 1 ]
set.add(5) // Set [ 1, 5 ]
set.add(5)

Array.from(set)
```

I [searched](https://bitbucket.org/search?account=%7B3c157d8e-1c3f-4d33-bbaf-0d83eb71672b%7D&q=addToArray%20ext%3Ajs%20-path%3A%2Fwww%20-path%3Autils) our client projects and found only one occurrence of this function.